File: dev/requirements/screen/cases/started-tally.4.4.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 4.4 — the list header counting what has been started out of what is planned.
let startedTally_4_4 = RequirementCase(id: "4.4") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "started-tally", id: "4.4")
    }
}
#endif

File: dev/requirements/screen/cases/started-tally.4.4.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 4.4 — The list header counts jokes started out of jokes planned.
final startedTally_4_4 = RequirementCase('4.4', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.setListCount,
    slug: 'started-tally',
    id: '4.4',
  );
});
